src/components/Contacts/ContactView/StatusSelector/StatusSelector.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, InputLabel, MenuItem, Select } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { ErrorOutline } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  statusLabel: {
+    padding: theme.spacing(0),
+    fontFamily: theme.typography.fontFamily,
+    color: theme.palette.text.primary,
```

Comment:
Are any of these CSS properties already the defaults? I think it's better to not specify them repeatedly unless you need to overwrite styles elsewhere.